pi-manage

```diff
@@ -586,6 +586,25 @@ def create(name, scope, action):
     return r
 
 
+@policy_manager.command
+def create_from_file(name, filename):
+    """
+    create a new policy from a JSON-file
+    """
+    import json
+
+    f = open(filename, 'r')
+    contents = f.read()
+    f.close()
+    params = json.loads(contents)
+
+    r = set_policy(name, scope=params["scope"], action=params["action"], realm=params["realm"],
+                   resolver=params["resolver"],
+                   user=params["user"], time=params["time"], client=params["client"], active=params["active"],
+                   adminrealm=params["adminrealm"], check_all_resolvers=params["check_all_resolvers"])
```

Comment:
Please use

    params.get("user")
    params.get("time")
    params.get("client")
    params.get("resolver")
    params.get("realm")
    params.get("active", True)
    params.get("check_all_resolvers": False)

You get the idea. So that you can simple create a file,where you can omit the keys time, client, user, active, adminrealm, check_all,... which are not often used and the defaults are fine.